WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/history/ScanHistoryListViewModel.kt

```diff
@@ -21,18 +26,28 @@ class ScanHistoryListViewModel @Inject constructor(
 
     lateinit var site: SiteModel
 
-    fun start(site: SiteModel, parentViewModel: ScanHistoryViewModel) {
+    fun start(tabType: ScanHistoryTabType, site: SiteModel, parentViewModel: ScanHistoryViewModel) {
         if (isStarted) return
         isStarted = true
         this.site = site
-        uiState = parentViewModel.threats.map { threats ->
-            // TODO malinjir filter by tab
-            threats.map {
-                scanThreatItemBuilder.buildThreatItem(it, this::onItemClicked)
-            }
-        }
+        uiState = parentViewModel
+                .threats
+                .map { threats ->
+                    threats
+                            .filter { mapTabTypeToThreatStatuses(tabType).contains(it.baseThreatModel.status) }
+                            .map {
+                                scanThreatItemBuilder.buildThreatItem(it, this::onItemClicked)
+                            }
```

Comment:
Minor (🔍): Consider formatting this a bit in order to save a couple of lines, like:
```
                .map { threats ->
                    threats.filter { mapTabTypeToThreatStatuses(tabType).contains(it.baseThreatModel.status) }
                            .map { scanThreatItemBuilder.buildThreatItem(it, this::onItemClicked) }
                }
```

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13827/commits/c203e55da6ebf1c4921c38bd862250836a134a9b